lxqt-admin-user/lxqt-admin-user-helper.freebsd

```diff
@@ -1,10 +1,5 @@
 #!/bin/sh
 case "$1" in
```

Comment:
As this is freebsd specific, we can easily change it to accept the same parameters as the \*.default one, but execute what is needed on freebsd => adding the `QStringLiteral("pw")` everywhere will not be needed.